src/util/type.h

```diff
@@ -54,7 +54,6 @@ class typet:public irept
     return static_cast<const typet &>(get_sub().front());
   }
 
-  // This method will be protected eventually.
```

Comment:
I don't think the commit message really matches what is happening here (the actual change is just the removal of a comment).

Reply:
Fixed